responses/pl/HassTurnOn.yaml

```diff
@@ -2,8 +2,9 @@ language: pl
 responses:
   intents:
     HassTurnOn:
-      default: Turned on {{ slots.name }}
-      lights_area: Turned on lights in {{ slots.area }}
-      fans_area: Turned on fans in {{ slots.area }}
-      cover: Opened {{ slots.name }}
-      cover_area: Opened {{ slots.area }}
+      default: Włączono {{ slots.name }}
+      lights_area: Włączono lampy w {{ slots.area }}
+      fan_area: Włączono wentylator w {{ slots.area }}
+      fans_area: Włączono wentylatory w {{ slots.area }}
+      cover: Otwarto {{ slots.name }}
```

Comment:
Zrobiłbym "Otworzono", choć obie formy są poprawne - https://sjp.pwn.pl/poradnia/haslo/otwarty-czy-otworzony;13903.html

Nie ukrywam, wolałbym usłyszeć z głośnika "Otworzono rolety" zamiast "Otwarto rolety", jeśli "otwarte" miałyby zostać to być może w formie: "rolety zostały otwarte" - jakoś tak lepiej brzmi. Co myślisz?

Reply:
We can't use sentence `rolety zostały otwarte` because for each sentence we have `{{ slots.name }}` or `{{ slots.area }}`. If we need use this approach we must use sentence `Rolety {{ slots.name }} zostały otwarte` and `Rolety w {{ slots.area }} zostały otwarte`.

Reply:
Second problem is, what happen if we try open one blind? In second approach `cover: Rolety {{ slots.name }} zostały otwarte` it is problematic because always ew use plural name..

Reply:
https://github.com/home-assistant/intents/issues/644

Reply:
How about supporting `opuść rolety` `podnieś rolet`?